// vuepress/lib/app/.temp/routes.js
import Vue from 'vue'
Vue.component("Homepage", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/Homepage.vue"))
Vue.component("MContent", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/MContent.vue"))
Vue.component("MarkdownCard", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/MarkdownCard.vue"))
Vue.component("Projects", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/Projects.vue"))
import ThemeLayout from '@themeLayout'
import ThemeNotFound from '@themeNotFound'
import { injectMixins } from '@app/util'
import rootMixins from '@app/root-mixins'

injectMixins(ThemeLayout, rootMixins)
injectMixins(ThemeNotFound, rootMixins)

export const routes = [
  {
    name: "v-4e504274b1feb",
    path: "/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/README.md").then(comp => {
        Vue.component("v-4e504274b1feb", comp.default)
        next()
      })
    }
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-0d546b2bada6e",
    path: "/article/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/article/README.md").then(comp => {
        Vue.component("v-0d546b2bada6e", comp.default)
        next()
      })
    }
  },
  {
    path: "/article/index.html",
    redirect: "/article/"
  },
  {
    name: "v-43539e00af2db",
    path: "/beasties/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/beasties/README.md").then(comp => {
        Vue.component("v-43539e00af2db", comp.default)
        next()
      })
    }
  },
  {
    path: "/beasties/index.html",
    redirect: "/beasties/"
  },
  {
    name: "v-d09717415f28d",
    path: "/beasties/old.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/beasties/old.md").then(comp => {
        Vue.component("v-d09717415f28d", comp.default)
        next()
      })
    }
  },
  {
    name: "v-8a6310803952e",
    path: "/murphy/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/README.md").then(comp => {
        Vue.component("v-8a6310803952e", comp.default)
        next()
      })
    }
  },
  {
    path: "/murphy/index.html",
    redirect: "/murphy/"
  },
  {
    name: "v-ad8953eefaf2e",
    path: "/murphy/gallery.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/gallery.md").then(comp => {
        Vue.component("v-ad8953eefaf2e", comp.default)
        next()
      })
    }
  },
  {
    name: "v-505686ef79acb",
    path: "/murphy/playlists.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/playlists.md").then(comp => {
        Vue.component("v-505686ef79acb", comp.default)
        next()
      })
    }
  },
  {
    name: "v-b2196a9f8d82c",
    path: "/murphy/recipes.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/recipes.md").then(comp => {
        Vue.component("v-b2196a9f8d82c", comp.default)
        next()
      })
    }
  },
  {
    name: "v-1cef79fd41c9",
    path: "/murphy/traditions.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/traditions.md").then(comp => {
        Vue.component("v-1cef79fd41c9", comp.default)
        next()
      })
    }
  },
  {
    path: '*',
    component: ThemeNotFound
  }
]
