// vuepress/lib/app/.temp/routes.js
import Vue from 'vue'
Vue.component("Homepage", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/Homepage.vue"))
Vue.component("MContent", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/MContent.vue"))
Vue.component("MarkdownCard", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/MarkdownCard.vue"))
Vue.component("Projects", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/Projects.vue"))
import ThemeLayout from '@themeLayout'
import ThemeNotFound from '@themeNotFound'
import { injectMixins } from '@app/util'
import rootMixins from '@app/root-mixins'

injectMixins(ThemeLayout, rootMixins)
injectMixins(ThemeNotFound, rootMixins)

export const routes = [
  {
    name: "v-06344581f78a8",
    path: "/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/README.md").then(comp => {
        Vue.component("v-06344581f78a8", comp.default)
        next()
      })
    }
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-6f9347e7dc1a",
    path: "/article/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/article/README.md").then(comp => {
        Vue.component("v-6f9347e7dc1a", comp.default)
        next()
      })
    }
  },
  {
    path: "/article/index.html",
    redirect: "/article/"
  },
  {
    name: "v-aa0ffd8751965",
    path: "/guide/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/guide/README.md").then(comp => {
        Vue.component("v-aa0ffd8751965", comp.default)
        next()
      })
    }
  },
  {
    path: "/guide/index.html",
    redirect: "/guide/"
  },
  {
    name: "v-c4fc8002c0b1",
    path: "/guide/advanced.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/guide/advanced.md").then(comp => {
        Vue.component("v-c4fc8002c0b1", comp.default)
        next()
      })
    }
  },
  {
    name: "v-7ee57a202c61d",
    path: "/guide/customize.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/guide/customize.md").then(comp => {
        Vue.component("v-7ee57a202c61d", comp.default)
        next()
      })
    }
  },
  {
    name: "v-85d86c7f83969",
    path: "/guide/getting-started.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/guide/getting-started.md").then(comp => {
        Vue.component("v-85d86c7f83969", comp.default)
        next()
      })
    }
  },
  {
    name: "v-10e1a5d7b99c6",
    path: "/projects/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/projects/README.md").then(comp => {
        Vue.component("v-10e1a5d7b99c6", comp.default)
        next()
      })
    }
  },
  {
    path: "/projects/index.html",
    redirect: "/projects/"
  },
  {
    name: "v-f4b4bec31a7be",
    path: "/projects/old.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/projects/old.md").then(comp => {
        Vue.component("v-f4b4bec31a7be", comp.default)
        next()
      })
    }
  },
  {
    path: '*',
    component: ThemeNotFound
  }
]
