// vuepress/lib/app/.temp/routes.js
import Vue from 'vue'
Vue.component("Homepage", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/Homepage.vue"))
Vue.component("MContent", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/MContent.vue"))
Vue.component("MarkdownCard", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/MarkdownCard.vue"))
Vue.component("Projects", () => import("/Users/teresagao/Desktop/beastsite/docs/.vuepress/components/Projects.vue"))
import ThemeLayout from '@themeLayout'
import ThemeNotFound from '@themeNotFound'
import { injectMixins } from '@app/util'
import rootMixins from '@app/root-mixins'

injectMixins(ThemeLayout, rootMixins)
injectMixins(ThemeNotFound, rootMixins)

export const routes = [
  {
    name: "v-7002e0abba138",
    path: "/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/README.md").then(comp => {
        Vue.component("v-7002e0abba138", comp.default)
        next()
      })
    }
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-2c588c3139e68",
    path: "/article/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/article/README.md").then(comp => {
        Vue.component("v-2c588c3139e68", comp.default)
        next()
      })
    }
  },
  {
    path: "/article/index.html",
    redirect: "/article/"
  },
  {
    name: "v-1515c6929f085",
    path: "/beasties/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/beasties/README.md").then(comp => {
        Vue.component("v-1515c6929f085", comp.default)
        next()
      })
    }
  },
  {
    path: "/beasties/index.html",
    redirect: "/beasties/"
  },
  {
    name: "v-ed60d9a85b1f5",
    path: "/beasties/old.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/beasties/old.md").then(comp => {
        Vue.component("v-ed60d9a85b1f5", comp.default)
        next()
      })
    }
  },
  {
    name: "v-59cdbd4371bcf",
    path: "/murphy/",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/README.md").then(comp => {
        Vue.component("v-59cdbd4371bcf", comp.default)
        next()
      })
    }
  },
  {
    path: "/murphy/index.html",
    redirect: "/murphy/"
  },
  {
    name: "v-a58db22241191",
    path: "/murphy/gallery.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/gallery.md").then(comp => {
        Vue.component("v-a58db22241191", comp.default)
        next()
      })
    }
  },
  {
    name: "v-1fd38fc727fd5",
    path: "/murphy/playlists.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/playlists.md").then(comp => {
        Vue.component("v-1fd38fc727fd5", comp.default)
        next()
      })
    }
  },
  {
    name: "v-87d4907b15e16",
    path: "/murphy/recipes.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/recipes.md").then(comp => {
        Vue.component("v-87d4907b15e16", comp.default)
        next()
      })
    }
  },
  {
    name: "v-91a228134af71",
    path: "/murphy/traditions.html",
    component: ThemeLayout,
    beforeEnter: (to, from, next) => {
      import("/Users/teresagao/Desktop/beastsite/docs/murphy/traditions.md").then(comp => {
        Vue.component("v-91a228134af71", comp.default)
        next()
      })
    }
  },
  {
    path: '*',
    component: ThemeNotFound
  }
]
